cindy/index.js

```diff
@@ -0,0 +1,106 @@
+// 38-PB-program-switch-multiplechoice
+// Switch if...
+// 1. Color Analyzer
+
+// Create a switch statement that prints a different statement for each color possibility (red, blue, green, yellow).
+
+function colors(myColor){
+    switch(myColor){
+  
+      case 1: 
+      case 2: 
+      case 3: return "Sexy red";
+      case 4: 
+      case 5: return 'Peaceful blue';
+      case 6: 
+      case 7: 
+      case 8: return 'Wild green';
+      case 9: 
+      case 10: 
+      case 11: 
+      case 12: return 'Funny yellow';
+      default: return " only from 1 - 12 :c";
+    }  }
+  console.log("choose your color according to your month of birth  => ", colors(13))
+
+
+  //2. Grading
+//Create a switch statement that prints different comments depending on a grade.
+function Grades(myGrade){
+    switch(myGrade){
+  
+      case 1: return "You Better work >:v"
+      case 2: return "Anything you can do I can do better."
+      case 3: return "It Wasn't Enough";
+      case 4: return "It Could Happen to You"
+      case 5: return 'Half The World Away';
+      case 6: return "Stayin' Alive"
+      case 7: return "A little messed up "
+      case 8: return 'Gimme Gimme Gimme';
+      case 9: return "September"
+      case 10: return 'First date'
+      default: return " There's no more than 10pts :c";
+    }  }
+  console.log("Take your fedback as a song  => ", Grades(10))
+
+  //3. Fruits
+//Create a switch statement that prints different statement for various fruit (e.g. banana, orange, strawberry, apple).
+function fruits(myFruit){
+    switch(myFruit){
+  
+      case 1: 
```

Comment:
Task3 ==> Please do the same like the first task, just delete empty cases

Reply:
done !